components/tabs/demo/custom-indicator.tsx

```diff
@@ -24,13 +24,29 @@ const items: TabsProps['items'] = [
   },
 ];
 
-const App: React.FC = () => (
-  <Tabs
-    defaultActiveKey="1"
-    items={items}
-    onChange={onChange}
-    indicatorSize={(origin) => origin - 16}
-  />
-);
+const App: React.FC = () => {
+  const [align, setAlign] = React.useState<TabsProps['indicatorAlign']>('center');
+  return (
+    <>
+      <Radio.Group
+        defaultValue="center"
+        style={{ marginBottom: 12 }}
+        onChange={(e) => setAlign(e.target.value)}
+        options={[
+          { label: 'start', value: 'start' },
+          { label: 'center', value: 'center' },
+          { label: 'end', value: 'end' },
+        ]}
```

Comment:
label 和 value 都一样可以直接 plain， 等用例跑完直接点应该可以？
```suggestion
        options={['start', 'center', 'end']}
```